File: C++/Easy/WrongAnswer/sum-of-all-odd-length-subarrays_0.cpp
class Solution {
public:
    int sumOddLengthSubarrays(vector<int>& arr) {
        
        int sum = 0;
        for (int i = 0; i < arr.size(); i++) {
            sum += arr[i] * ((i + 1) * (arr.size() - i) + 1) / 2;
        }
        return sum;
    }
};

File: Rust/Easy/Accepted/sum-of-all-odd-length-subarrays_0.rs
impl Solution {
    pub fn sum_odd_length_subarrays(arr: Vec<i32>) -> i32 {
        
        let mut sum = 0;
        for i in 0..arr.len() {
            for j in i..arr.len() {
                if (j - i + 1) % 2 == 1 {
                    sum += arr[i..=j].iter().sum::<i32>();
                }
            }
        }
        sum
    }
}